examples/http-access-logs/Config.toml

```diff
@@ -1,5 +1,9 @@
 [ballerina.http.accessLogConfig]
 # Enable printing access logs on the Console. The default value is `false`.
 console = true
+# Specify the format of access logs. Options are `flat` or `json`. The default value is `flat`.
+format = "flat"
+# List of attributes to include in the access logs. This field is optional.
+attributes = ["ip", "date_time", "request", "status", "response_body_size", "http_referrer", "http_user_agent"]
```

Comment:
Shall we describe what the behavior is when we add the `attributes=[]` or no attributes field. Wdyt?

Reply:
@AzeemMuzammil  It is given in the spec right?

Reply:
Yes, the values that are supported part of the attributes are specified in the spec